package.json

```diff
@@ -61,19 +79,6 @@
     "url": "https://github.com/upstash/upstash-redis/issues"
   },
   "homepage": "https://github.com/upstash/upstash-redis#readme",
-  "typesVersions": {
```

Comment:
Do we have any differences between type exports? Maybe we should add the type exports to `exports` map, now that we've removed it from here.

Reply:
For cf and fastly exports here, typescript finds the appropriate .d.ts file itself, we don't have to specify it.

And for the `nodejs` here, I checked and it does not work in the prod version so we don't need it here